app/Http/Controllers/LoginController.php

```diff
@@ -70,4 +70,12 @@ public function logout()
        auth()->logout();
 
     }
+
+    public function sortAuthenticatedUsers()
```

Comment:
This same code block exists in both the login and user group controller, is it needed in both?